BWEnv/src/controller.cc

```diff
@@ -346,37 +357,44 @@ void Controller::handleCommand(int command, const std::vector<int>& args,
         args[0], cmd_type, target,
         x, y, extra);
       if (!unit->issueCommand(BWAPI::UnitCommand(unit, cmd_type, target,
-        x, y, extra))
-        ) Utils::bwlog(output_log, "Commanding unit failed! Error: %s",
-        BWAPI::Broodwar->getLastError().c_str());
-      return;
+        x, y, extra))) {
+        Utils::bwlog(output_log, "Commanding unit failed! Error: %s",
+          BWAPI::Broodwar->getLastError().c_str());
+        return CommandStatus::BWAPI_ERROR;
```

Comment:
Is it useful to return the type of BWAPI_ERROR here as well? There's actually only a few errors you can make anyway: https://bwapi.github.io/namespace_b_w_a_p_i_1_1_errors.html